packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -708,7 +723,7 @@ export class Cluster extends Resource implements ICluster {
    * @param id the id of this service account
    * @param options service account options
    */
-  public addServiceAccount(id: string, options: ServiceAccountOptions) {
+  public addServiceAccount(id: string, options: ServiceAccountOptions = { }) {
```

Comment:
the `options` is still mandatory here

maybe

```
   public addServiceAccount(id: string, options?: ServiceAccountOptions = { }) { 
```

Reply:
The default value turns it into optional. The “?” Is not needed